build/dependencies.yaml

```diff
@@ -99,22 +99,22 @@ dependencies:
 
   # Golang
   - name: "golang: upstream version"
-    version: 1.15.2
+    version: 1.15.5
     refPaths:
     - path: build/build-image/cross/VERSION
     - path: build/root/WORKSPACE
       match: (override_)?go_version = "\d+.\d+(alpha|beta|rc)?\.?\d+"
+    - path: cluster/addons/fluentd-elasticsearch/es-image/Dockerfile
+      match: 'FROM golang\:\d+.\d+(alpha|beta|rc)?\.?(\d+)?'
     - path: test/images/Makefile
       match: GOLANG_VERSION=\d+.\d+(alpha|beta|rc)?\.?\d+
     - path: staging/publishing/rules.yaml
       match: 'default-go-version\: \d+.\d+(alpha|beta|rc)?\.?(\d+)?'
 
   - name: "k8s.gcr.io/kube-cross: dependents"
-    version: v1.15.2-1
+    version: v1.15.5-1
     refPaths:
     - path: build/build-image/cross/VERSION
-    - path: test/images/sample-apiserver/Makefile
```

Comment:
curious why this got dropped

Reply:
@liggitt -- the sample apiserver uses a different image now and the reference no longer exists: https://github.com/kubernetes/kubernetes/blob/0c973409051faa8926971cfb05b967a2d88c5ca8/test/images/sample-apiserver/BASEIMAGE#L1

Reply:
ack

Reply:
actually, is that true? I still see `k8s.gcr.io/build-image/kube-cross:v1.15.2-1` at https://github.com/kubernetes/kubernetes/blob/master/test/images/sample-apiserver/Makefile#L27